web/src/message_util.ts

```diff
@@ -1,8 +1,12 @@
+import assert from "minimalistic-assert";
+
 import {all_messages_data} from "./all_messages_data";
 import type {MessageListData} from "./message_list_data";
-import type {MessageList, RenderInfo} from "./message_lists";
+import {type MessageList, type RenderInfo, current} from "./message_lists";
 import * as message_store from "./message_store";
 import type {Message} from "./message_store";
+import * as people from "./people";
+import {recent} from "./pm_conversations";
```

Comment:
Only import types without the module name, both here and with `current`. It's really helpful to be able to `git grep pm_conversations[.]recent` and find all the code that uses something.